.env.local

```diff
@@ -23,3 +23,14 @@ MUMBAI_LENSAPI_ORACLE_ENDPOINT=0xa31cb575847033341a1af63ac86fee33818a4838
 # for Polygon Mumbai testnet deployment
 POLYGON_CONSUMER_CONTRACT_ADDRESS=0xad5C96e026D4D00c9a6Fcd1AA2E4958a5155AD03
 MUMBAI_CONSUMER_CONTRACT_ADDRESS=0x5462964078808abc96f7F45Ff7f3158B819cD7E3
+
+# Don't touch it if you don't know what it is.
+PHALA_MAINNET_ENDPOINT=wss://api.phala.network/ws
+PHALA_TESTNET_ENDPOINT=wss://poc5.phala.network/ws
+
+# Don't touch it if you don't know what it is.
+PHAT_BRICKS_MAINNET_FACTORY_CONTRACT_ID=0xb59bcc4ea352f3d878874d8f496fb093bdf362fa59d6e577c075f41cd7c84924
+PHAT_BRICKS_TESTNET_FACTORY_CONTRACT_ID=0x489bb4fa807bbe0f877ed46be8646867a8d16ec58add141977c4bd19b0237091
+
+# The encypt passphrase for the exported polkadot wallet JSON file.
+POLKADOT_WALLET_PASSPHRASE=
```

Comment:
Suggest to allow loading mnemonic (sURI) by default. Json keystore should be the backup option.